src/nist.gov/SP800-53/rev5/xml/NIST_SP-800-53_rev5_catalog.xml

```diff
@@ -83249,5 +83619,9 @@
       <title>NIST Special Publication 800-53, Revision 5: <em> Security and Privacy Controls for Information Systems and Organizations</em> (DOI link)</title>
       <rlink media-type="application/pdf" href="https://doi.org/10.6028/NIST.SP.800-53r5"/>
     </resource>
+    <resource uuid="5a5ffcb9-2272-484e-8d47-4483a0585dec">
+      <title>NIST OSCAL GitHub Version Control System for SP 800-53 Revision 5 Catalogs</title>
```

Comment:
"... for SP 800-53 Revision 5.1.1" <- to reflect the NIST RMF team's version?

Reply:
It contains multiple versions of the catalog, it is this repository (so it has versions of 5.0.0, 5.0.1) it even points to the Revision 4, but if this is what you want fine.

Reply:
I updated it to match the "title" in the about of our GitHub repo.